docs/src/testing/testing.md

```diff
@@ -41,7 +41,88 @@ test case.
 Starknet Forge will collect tests only from these places:
 
 - any files reachable from the package root (declared as `mod` in `lib.cairo` or its children)
-- files inside the `tests` directory
+- files inside the [`tests`](#the-tests-directory) directory
+
+## The *tests* Directory
+
+### With *tests/lib.cairo*
```

Comment:
I'd write something like: `Forge collects tests from `tests` directory. Depending on if `lib.cairo` file is defined there, the behavior of the test collector will be different.`